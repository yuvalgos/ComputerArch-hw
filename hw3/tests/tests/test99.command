./dflow_calc tests/test99.dat tests/test99.in d81 p52 p142 p71 p48 d81 p178 p189 p3 p144 p0 d177 d99 p104 d33 d127 p150 d3 p16 d43 d117 d179 p64 p106 d193 p86 p44 p143 p78 d45 p59 p99 d66 p140 d13 d101 p114 d35 p145 d198 p182 d71 p128 d168 d6 d156 d190 p151 d44 d151 d25 p41 p116 d10 d63 d119 p28 p34 d57 d100 d16 p58 d84 p196 p134 p17 p24 p182 d145 d94 d94 d195 d83 d76 d154 d189 d68 d54 p163 p157 d39 p25 p81 d29 d58 p35 d130 p74 d70 d116 d1 d126 d18 d163 d38 d168 d21 p68 p56 d26 p168 p3