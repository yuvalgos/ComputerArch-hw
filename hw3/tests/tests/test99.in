6 5 26 13
0 18 24 4
7 16 26 6
5 5 29 6
5 9 13 2
6 16 5 26
0 19 15 10
8 30 22 1
8 21 28 11
2 8 27 15
5 31 24 22
1 17 25 2
5 4 11 25
2 12 7 8
4 12 19 11
7 2 15 28
0 28 0 20
8 3 13 21
4 11 28 23
6 10 16 3
4 11 11 22
8 20 28 3
4 6 16 5
3 4 30 2
3 16 24 3
6 16 7 14
7 19 28 27
6 3 25 30
0 28 12 25
6 17 28 9
5 28 18 3
0 22 26 9
6 30 17 31
0 1 22 15
4 28 26 1
8 19 23 14
8 18 10 16
8 14 8 26
4 16 24 12
7 24 9 20
0 7 14 27
2 23 2 12
7 28 9 15
8 3 23 5
5 21 8 7
6 10 21 31
4 9 13 25
6 8 13 18
3 17 31 28
4 19 31 7
3 26 25 23
8 2 5 21
2 22 10 9
3 20 12 31
0 18 2 16
5 27 1 30
4 21 6 11
6 19 29 25
1 13 13 22
3 22 17 10
7 15 31 3
1 29 21 17
4 5 15 17
7 16 22 16
3 9 12 15
2 16 1 0
3 5 26 24
2 9 15 8
8 1 14 29
6 10 6 15
4 9 9 26
6 3 14 13
8 29 31 15
2 23 21 23
7 26 23 11
3 29 14 19
8 27 15 9
4 25 19 31
5 0 16 8
7 7 5 28
2 14 17 31
3 1 31 22
8 10 8 20
6 6 2 16
4 24 21 30
5 11 4 9
8 12 30 13
8 17 31 26
3 20 5 18
1 14 16 6
5 26 2 21
4 30 14 18
8 31 19 13
1 23 29 3
5 25 17 29
8 7 21 13
2 22 27 27
3 13 16 14
3 27 8 3
4 21 26 8
8 7 4 31
4 13 22 16
3 22 29 22
3 29 25 30
0 27 28 17
1 27 11 30
8 13 8 27
5 28 15 18
0 4 11 0
6 3 3 14
4 20 9 1
0 12 24 30
3 25 8 17
1 22 0 13
6 15 1 28
1 21 25 13
7 30 0 28
4 23 11 18
6 27 16 0
5 19 4 27
2 25 22 25
3 7 0 25
7 18 17 10
7 24 9 13
5 24 11 22
0 10 2 1
0 7 10 29
3 29 7 31
1 5 28 22
2 17 9 16
4 12 17 19
4 1 20 5
5 16 13 10
2 11 25 25
1 28 5 27
5 30 7 10
6 17 11 10
8 15 31 27
3 11 31 26
1 21 24 20
0 1 26 26
5 6 16 11
8 18 19 0
3 16 11 5
5 9 1 17
3 6 16 16
4 17 29 17
5 9 15 12
1 30 4 16
2 19 25 19
1 20 27 14
3 17 25 12
2 29 15 5
7 27 15 14
0 12 13 26
3 18 15 26
2 5 15 1
3 25 15 27
3 11 8 10
7 1 14 9
2 15 25 25
0 3 9 15
1 2 22 17
6 22 5 15
6 22 14 20
4 12 22 7
5 26 10 19
3 1 13 11
6 22 29 4
1 10 20 29
3 15 13 1
4 18 12 6
0 12 24 12
3 7 31 14
3 17 6 10
7 19 10 30
7 13 1 2
5 30 17 23
0 5 15 19
6 23 6 16
0 20 5 29
2 23 30 26
1 15 13 9
5 25 7 15
1 25 20 2
4 22 20 27
5 29 30 30
4 25 20 10
1 2 4 14
0 2 0 29
5 18 30 6
3 11 11 8
0 11 16 14
5 27 2 15
5 22 26 26
5 3 4 26
1 10 7 27
5 24 21 20
4 8 1 18
4 9 16 30
