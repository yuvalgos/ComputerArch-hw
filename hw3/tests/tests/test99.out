Reading the opcodes latency info from tests/test99.dat ... Got latency for 9 opcodes
Reading the program file tests/test99.in ... Found 200 instructions
getProgDepth()==229
getInstDeps(81)=={10,59}
getDepDepth(52)==60
getDepDepth(142)==147
getDepDepth(71)==68
getDepDepth(48)==32
getInstDeps(81)=={10,59}
getDepDepth(178)==193
getDepDepth(189)==211
getDepDepth(3)==0
getDepDepth(144)==188
getDepDepth(0)==0
getInstDeps(177)=={174,117}
getInstDeps(99)=={90,47}
getDepDepth(104)==99
getInstDeps(33)=={31,-1}
getInstDeps(127)=={126,92}
getDepDepth(150)==133
getInstDeps(3)=={-1,-1}
getDepDepth(16)==0
getInstDeps(43)=={41,3}
getInstDeps(117)=={85,54}
getInstDeps(179)=={145,143}
getDepDepth(64)==66
getDepDepth(106)==135
getInstDeps(193)=={189,182}
getDepDepth(86)==68
getDepDepth(44)==30
getDepDepth(143)==164
getDepDepth(78)==103
getInstDeps(45)=={44,10}
getDepDepth(59)==60
getDepDepth(99)==103
getInstDeps(66)=={50,39}
getDepDepth(140)==79
getInstDeps(13)=={-1,9}
getInstDeps(101)=={96,65}
getDepDepth(114)==81
getInstDeps(35)=={-1,-1}
getDepDepth(145)==171
getInstDeps(198)=={167,190}
getDepDepth(182)==195
getInstDeps(71)=={37,58}
getDepDepth(128)==161
getInstDeps(168)=={152,108}
getInstDeps(6)=={-1,-1}
getInstDeps(156)=={137,140}
getInstDeps(190)=={177,145}
getDepDepth(151)==166
getInstDeps(44)=={9,40}
getInstDeps(151)=={120,130}
getInstDeps(25)=={-1,-1}
getDepDepth(41)==19
getDepDepth(116)==116
getInstDeps(10)=={-1,-1}
getInstDeps(63)=={59,38}
getInstDeps(119)=={108,118}
getDepDepth(28)==13
getDepDepth(34)==17
getInstDeps(57)=={-1,-1}
getInstDeps(100)=={23,92}
getInstDeps(16)=={-1,-1}
getDepDepth(58)==65
getInstDeps(84)=={56,32}
getDepDepth(196)==226
getDepDepth(134)==164
getDepDepth(17)==23
getDepDepth(24)==46
getDepDepth(182)==195
getInstDeps(145)=={143,143}
getInstDeps(94)=={87,75}
getInstDeps(94)=={87,75}
getInstDeps(195)=={108,166}
getInstDeps(83)=={51,65}
getInstDeps(76)=={60,70}
getInstDeps(154)=={106,90}
getInstDeps(189)=={78,186}
getInstDeps(68)=={37,61}
getInstDeps(54)=={51,38}
getDepDepth(163)==187
getDepDepth(157)==175
getInstDeps(39)=={4,21}
getDepDepth(25)==0
getDepDepth(81)==67
getInstDeps(29)=={28,4}
getInstDeps(58)=={-1,52}
getDepDepth(35)==0
getInstDeps(130)=={129,119}
getDepDepth(74)==57
getInstDeps(70)=={67,50}
getInstDeps(116)=={78,107}
getInstDeps(1)=={-1,-1}
getInstDeps(126)=={125,103}
getInstDeps(18)=={16,-1}
getInstDeps(163)=={156,160}
getInstDeps(38)=={-1,14}
getInstDeps(168)=={152,108}
getInstDeps(21)=={16,17}
getDepDepth(68)==55
getDepDepth(56)==41
getInstDeps(26)=={16,-1}
getDepDepth(168)==169
getDepDepth(3)==0
